vendor/github.com/vmware/alb-sdk/go/models/virtual_service.go

```diff
@@ -113,7 +113,7 @@ type VirtualService struct {
 	HTTPPolicies []*HTTPPolicies `json:"http_policies,omitempty"`
 
 	// The config settings for the ICAP server when checking the HTTP request. It is a reference to an object of type IcapProfile. Field introduced in 20.1.1. Maximum of 1 items allowed. Allowed in Enterprise edition with any value, Enterprise with Cloud Services edition.
-	IcapRequestProfileRefs []string `json:"icap_request_profile_refs,omitempty"`
+	IcapRequestProfileRefs *[]string `json:"icap_request_profile_refs,omitempty"`
```

Comment:
Is this change done by mistake? Ideally, we won't make any changes to files in the vendor folders.

Reply:
I wasn't aware of that, will revert it back and change all references accordingly.